dm/dm/master/server.go

```diff
@@ -2658,7 +2674,15 @@ func (s *Server) StopValidation(ctx context.Context, req *pb.StopValidationReque
 		// nolint:nilerr
 		return resp, nil
 	}
-	// TODO: update the validator stage in etcd
+	err = s.scheduler.OperateValidationTask(pb.Stage_Stopped, subTaskCfgs)
```

Comment:
seems `IsAllTask` has been forgotten 🤔

Reply:
`IsAllTask` is handled by `scheduler.GetSubTaskCfgsByTaskAndSource(taskName, sources)`, where the `taskName` must be empty and this function will return all the tasks that matches the `sources`. The returned subtasks are in `subTaskCfgs`.

Reply:
so if we list all tasks when  `req.Task ==""` like `dmctl query-status`, we can save one request param(req. IsAllTask) ?

Reply:
Indeed. I'll fix it.